### PR TITLE
Modulereview

### PR DESCRIPTION
**Pull request by Jesse132**
Modified overview and implementation of 'sticky buttons' to dark mode

### CODE REVIEW

File: client/src/components/Overview/ImageShowcase.jsx [hunk 1]
```diff
@@ -83,6 +91,7 @@ const ImageShowcase = ({productStyle, thumbnailChange, selectedThumbnailIndex})
               className={`slideshowDot${selectedThumbnailIndex === idx ? ' active' : ''}`}
               onClick={() => {
                 thumbnailChange(idx);
+                console.log(idx);
```
Comment (poioper) [inline client/src/components/Overview/ImageShowcase.jsx:94]: Could you please remove this?

File: client/src/components/styles/Overview/ImageShowcase.styled.js [hunk 2]
```diff
@@ -75,6 +75,44 @@ export const StyledImgShowcase = styled.div`
     background-color: #000000;
   }
 
+  .leftArrow {
```
Comment (poioper) [inline client/src/components/styles/Overview/ImageShowcase.styled.js:78]: Are the leftArrow rightArrow still necessary?

**Review by poioper — APPROVED** [review]
Looks great! May need to delete some extra content

**Review by codingavatar — APPROVED** [review]
Looks great. I will make the deletes on my end for the next PR and merge

**Pull request merged**